src/pages/table.js

```diff
@@ -1,61 +1,84 @@
 import React from 'react';
-import { Button, Checkbox, Icon, Table, Segment } from 'semantic-ui-react';
+import { Button, Checkbox, Icon, Table, Segment, Input } from 'semantic-ui-react';
 import 'semantic-ui-css/semantic.min.css'
+import { getAssistantsForEvent, editAssistant } from '../firebase';
+
+const TableRow = ({ assistant, eventId, firebase }) => {
+  const [isEditable, setIsEditable] = React.useState(false);
+  const [formData, setFormData] = React.useState({
+    ...assistant
+  })
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value })
+  }
+  return (
+    !isEditable ? (
+      <Table.Row>
+        <Table.Cell collapsing>
+          <Checkbox slider defaultChecked={formData.attended} onChange={(event, { checked }) => { editAssistant(firebase, { ...formData, attended: checked }, eventId) }} />
+        </Table.Cell>
+        <Table.Cell>{formData.firstName}</Table.Cell>
+        <Table.Cell>{formData.lastName}</Table.Cell>
+        <Table.Cell>{formData.curp}</Table.Cell>
+        <Table.Cell>{formData.phone}</Table.Cell>
+        <Table.Cell>{formData.cellphone}</Table.Cell>
+        <Table.Cell>{formData.email}</Table.Cell>
+        <Table.Cell><button className="ui button" onClick={() => setIsEditable(true)}>Editar</button></Table.Cell>
+      </Table.Row>
+    ) : (
+        <Table.Row>
+          <Table.Cell collapsing>
+            <Checkbox slider />
+          </Table.Cell>
+          <Table.Cell><Input name="firstName" defaultValue={formData.firstName} onChange={handleChange} /></Table.Cell>
+          <Table.Cell><Input name="lastName" value={formData.lastName} onChange={handleChange} /></Table.Cell>
+          <Table.Cell><Input name="curp" value={formData.curp} onChange={handleChange} /></Table.Cell>
+          <Table.Cell><Input name="phone" value={formData.phone} onChange={handleChange} /></Table.Cell>
+          <Table.Cell><Input name="cellphone" value={formData.cellphone} onChange={handleChange} /></Table.Cell>
+          <Table.Cell><Input name="email" value={formData.email} onChange={handleChange} /></Table.Cell>
+          <Table.Cell>
+            <button type="submit" className="ui button" onClick={() => { editAssistant(firebase, formData, eventId); setIsEditable(false) }}>Guardar</button><button className="ui button" onClick={() => setIsEditable(false)}>Cancelar</button>
```

Comment:
demasiado para una sola línea. no se alcanza a leer lo que es. 

```suggestion
            <button type="submit" className="ui button" onClick={() => { editAssistant(firebase, formData, eventId); setIsEditable(false) }}>
                Guardar
            </button>
            <button className="ui button" onClick={() => setIsEditable(false)}>
                Cancelar
            </button>
```
Y mover la funcion del primer boton a una variable.